packages/hs-react-ui/src/components/Table/Table.tsx

```diff
@@ -1,48 +1,83 @@
 import React, { useState, useEffect } from 'react';
 import styled, { StyledComponentBase } from 'styled-components';
 import useResizeObserver from 'use-resize-observer';
+import { mdiArrowDown, mdiChevronRight, mdiChevronDown, mdiChevronUp } from '@mdi/js';
 import Icon from '@mdi/react';
-import { mdiArrowDown } from '@mdi/js';
-
 import colors from '../../enums/colors';
 import fonts from '../../enums/fonts';
 
+/** Types and Interfaces */
+
+export type CellOptions = {
+  RenderedCell: any;
+  headerColumnKey: string; //
+  breakPointHit: boolean;
+  row: columnTypes;
+  index: number;
+  indexModifier?: number;
+  CollapseIcon?: any;
+  groupIndex?: number;
+  isCollapsed?: boolean;
+  cIcon?: any;
+  eIcon?: any;
```

Comment:
what are these for? Collapsed/Expanded icons? If so, the prevoius CollapseIcon prop is probably a duplicate? Either way we should use full prop names like CollapseIcon to make them easy to understand

Reply:
These are the overrides for the expanded and collapsed icons. It's how a developer can pass material Icons (or any svg path) of their choice into the Table without providing their own custom expansion Icon too.